stdlib/2.7/select.pyi

```diff
@@ -66,7 +66,7 @@ POLLWRBAND = ...  # type: int
 POLLWRNORM = ...  # type: int
 
 def poll() -> epoll: ...
-def select(rlist, wlist, xlist, timeout: Optional[int]) -> Tuple[List, List, List]: ...
+def select(rlist, wlist, xlist, timeout: int = None) -> Tuple[List, List, List]: ...
```

Comment:
Actually the timeout is a float. Can you change that too? (There's more -- I think the Python 3 stub is a bit more complete, but that cal all wait.)